### PR TITLE
EN-4110: reward tx preprocessor unit tests

### PR DESCRIPTION
**Pull request by bogdan-rosianu**

### CODE REVIEW

File: process/block/preprocess/rewardTxPreProcessor_test.go [hunk 1]
```diff
@@ -0,0 +1,742 @@
...
+	if _, ok := res[txHash]; !ok {
+		assert.Fail(t, "miniblock was not added")
+	}
+}
+
+func TestRewardTxPreprocessor_CreateMarshalizedDataShouldWork(t *testing.T) {
+	t.Parallel()
+
+	txHash := "tx1_hash"
+
```
Comment (sasurobert) [inline process/block/preprocess/rewardTxPreProcessor_test.go]: delete empty line

File: process/block/preprocess/rewardTxPreProcessor_test.go [hunk 2]
```diff
@@ -0,0 +1,742 @@
...
+	tdp := initDataPool()
+
```
Comment (sasurobert) [inline process/block/preprocess/rewardTxPreProcessor_test.go]: delete empty line

File: process/block/preprocess/rewardTxPreProcessor_test.go [hunk 3]
```diff
@@ -0,0 +1,742 @@
...
+		&mock.HasherMock{},
+		&mock.MarshalizerMock{},
+		&mock.RewardTxProcessorMock{},
+		&mock.IntermediateTransactionHandlerMock{},
+		mock.NewMultiShardsCoordinatorMock(3),
+		&mock.AccountsStub{},
+		func(shardID uint32, txHashes [][]byte) {},
+	)
+
+	assert.NotNil(t, rtp)
```
Comment (sasurobert) [inline process/block/preprocess/rewardTxPreProcessor_test.go]: no need for this assert.

File: process/block/preprocess/rewardTxPreProcessor_test.go [hunk 4]
```diff
@@ -0,0 +1,742 @@
...
+
+	txHashes := [][]byte{[]byte(txHash)}
+
```
Comment (sasurobert) [inline process/block/preprocess/rewardTxPreProcessor_test.go]: delete empty line

File: process/block/preprocess/rewardTxPreProcessor_test.go (hunk 2 above)
Comment (bogdan-rosianu) [reply process/block/preprocess/rewardTxPreProcessor_test.go]: done

File: process/block/preprocess/rewardTxPreProcessor_test.go (hunk 3 above)
Comment (bogdan-rosianu) [reply process/block/preprocess/rewardTxPreProcessor_test.go]: done

File: process/block/preprocess/rewardTxPreProcessor_test.go (hunk 4 above)
Comment (bogdan-rosianu) [reply process/block/preprocess/rewardTxPreProcessor_test.go]: done

**Pull request merged**